third_party/2and3/werkzeug/exceptions.pyi

```diff
@@ -157,3 +157,5 @@ class Aborter:
     def __call__(self, code: Union[int, Response], *args: Any, **kwargs: Any) -> NoReturn: ...
 
 def abort(status: Union[int, Response], *args: Any, **kwargs: Any) -> NoReturn: ...
+
+BadRequestKeyError: BadRequest
```

Comment:
I think you need a new class inheriting from both BadRequest and KeyError (see the implementation of wrap: https://github.com/pallets/werkzeug/blob/a220671d66755a94630a212378754bb432811158/src/werkzeug/exceptions.py#L93).

Reply:
That makes sense. Would the following work?

```python
class BadRequestKeyError(BadRequest, KeyError): ...
```

I'm not sure if it's necessary to put the other attributes/functions of the exception? I'm guessing they would get picked up by the type checker from the fact that this is a subclass of `BadRequest`?

Reply:
Correct, from the code I don't see any attributes it has that its parent classes don't have.

Reply:
Excellent. I updated the commit with the correct implementation.